duckduckgo/content-scope-scripts
# ntp: customizer drawer screens

**Asana Task/Github Issue:** 

## Description

- [x] added a baseline implementation of every UI interaction (minus animations and polish)
- [x] designed for early merge, so the changes SHOULD NOT impact the existing NTP (the one without the drawer enabled)

## Testing Steps

- When customizer draw is DISABLED:
  - Visit [this link](https://deploy-preview-1303--content-scope-scripts.netlify.app/build/pages/new-tab/?customizerDrawer=disabled)
  - Ensure light+dark mode is still working as expected
  - Please test as many variants of this page as you know of (eg: more/less elements)
  
- When customizer draw is ENABLED:
  - We don’t need to verify anything here yet, but you can view the WIP by setting the drawer to ‘Enabled’ https://deploy-preview-1303--content-scope-scripts.netlify.app/build/pages/new-tab/?customizerDrawer=disabled

## Checklist


*Please tick all that apply:*
